lib/public/views/Runs/Overview/RunsOverviewModel.js

```diff
@@ -51,81 +52,59 @@ export class RunsOverviewModel extends Observable {
         this._eorReasonsFilterModel.observe(() => this._applyFilters());
         this._eorReasonsFilterModel.visualChange$.observe(() => this.notify());
 
-        this._pagination = new PaginationModel();
-        this._pagination.observe(() => this.fetchRuns());
-        this._pagination.itemsPerPageSelector$.observe(() => this.notify());
-
-        // Content
-        this._currentPageRuns = RemoteData.notAsked();
-        this._allRuns = RemoteData.notAsked();
+        // Exportable items
+        this._exportableDataSource = new PaginatedRemoteDataSource();
```

Comment:
Why is it paginated?

Reply:
bad idea, change to just `RemoteDataSource` with proper `RemoteData#apply`